src/manifest.c

```diff
@@ -574,29 +574,82 @@ static void set_untracked_manifest_files(struct manifest *manifest)
 	}
 }
 
-//NOTE: file==NULL when component=="MoM", else file->hash is needed
-int load_manifests(int current, int version, char *component, struct file *file, struct manifest **manifest)
+/* Loads the MoM (Manifest of Manifests) for VERSION.
+ * MOM manifest.
+ *
+ * Value ranges and restrictions:
+ *   - CURRENT < VERSION for 'update' subcommand
+ *   - CURRENT == VERSION for other subcommands
+ *
+ * Note that if the manifest fails to download, or if the manifest fails to be
+ * loaded into memory, this function will return NULL.
+ */
+struct manifest *load_mom(int current, int version)
 {
+	struct manifest *manifest = NULL;
 	int ret = 0;
 
-	*manifest = manifest_from_file(version, component);
+	ret = retrieve_manifests(current, version, "MoM", NULL);
```

Comment:
- ret = retrieve_manifests(current, version, "MoM", NULL);
+ ret = retrieve_manifests(version, version, "MoM", NULL);

If called in this way, then the NULL and version not less than version result in retrieve_manifests() not attempting a delta application.